### PR TITLE
Cleanup logging

### PR DESCRIPTION
**Pull request by mellenburg**
High level description including:
- Many of our builds logs are extremely hard to parse
- Test logs should pump out status logging info continuously so sudden failure causes can be diagnosed

### CODE REVIEW

**Comment by mellenburg** [comment]
@cmaloney This is ready for review. I watched the logs as the tests were running to confirm that info is streaming out now. Thus, if the cluster fails to come up, one will finally be able to see the cause in the logs

File: ssh/ssh_tunnel.py [hunk 1]
```diff
@@ -88,13 +88,13 @@ def write_to_remote(self, src, dst):
...
         cmd = self.ssh_cmd + ['-C', '-p', str(self.port), self.target, 'cat>' + dst]
-        logger.debug('Running socket write: ' + ' '.join(cmd))
+        log.info('Running socket write: ' + ' '.join(cmd))
         with open(src, 'r') as fh:
             check_call(cmd, stdin=fh)
 
     def close(self):
         close_tunnel = self.ssh_cmd + ['-p', str(self.port), '-O', 'exit', self.target]
-        logger.debug('Closing SSH Tunnel: ' + ' '.join(close_tunnel))
+        log.debug('Closing SSH Tunnel: ' + ' '.join(close_tunnel))
```
Comment (cmaloney) [inline ssh/ssh_tunnel.py]: I think this should become log.info() as well. We log.info() the creation, and if this happens unexpectedly early bad things happen.

File: test_util/azure_test_driver.py [hunk 2]
```diff
@@ -17,6 +18,10 @@
 from ssh.ssh_tunnel import SSHTunnel
 from test_util.test_runner import integration_test
 
+logging.getLogger("msrest").setLevel(logging.INFO)
+logging.getLogger("msrest").setLevel(logging.INFO)
```
Comment (cmaloney) [inline test_util/azure_test_driver.py]: duplicate line

**Comment by cmaloney** [comment]
Currently this has both `-vv -s -rs` and `-rs -s -vv`. Can we make it only do one of them?

**Comment by mellenburg** [comment]
@cmaloney This PR has been updated so that every test now calls 'pytest -vv -s -rs'

**Comment by cmaloney** [comment]
:ship:
